### PR TITLE
Feature/mortar 67

### PR DESCRIPTION
**Pull request by vamidjo**
https://jira.nationalgeographic.com/browse/MORTAR-67

Test instructions: 

- Git checkout
- Gulp serve
- Visit http://localhost:3000/patterns

### CODE REVIEW

File: app/modules/modals/_styles.scss [hunk 1]
```diff
@@ -17,6 +17,7 @@
   top: 7.5vh;
   transition: .2s;
   width: 85vw;
+  z-index: 999;
```
Comment (xiwcx) [inline app/modules/modals/_styles.scss]: z values should all be variables. make a new file: `global/vars/_z.scss`

it should contain

```scss
$z-bottom: -1;
$z-middle: 0;
$z-top: 999;
```

then reference that variable.

File: app/styles/_shame.scss [hunk 2]
```diff
@@ -238,10 +239,10 @@ td, th {
 }
 
 .dashed-border {
-  background-image: -webkit-linear-gradient(top,$mt_color-primary 50%,transparent 0);
-  background-image: linear-gradient(to bottom,$mt_color-primary 50%,transparent 0);
+  background-image: -webkit-linear-gradient(top, $mt_color-primary 50%,transparent 0);
```
Comment (xiwcx) [inline app/styles/_shame.scss]: did autoprefixer miss this? if not, you shouldn't need the vendor prefixes.

File: app/styles/blocks/_counters.scss [hunk 3]
```diff
@@ -14,13 +14,20 @@
 }
 
 .dotcounter-container {
```
Comment (xiwcx) [inline app/styles/blocks/_counters.scss]: missint `.mt_`

File: app/styles/blocks/_videopromo.scss [hunk 4]
```diff
@@ -10,15 +10,15 @@
   background-size: contain;
   background-image: $mt_video-promo-play-button;
   border: none;
+  content: $mt_unit;
```
Comment (xiwcx) [inline app/styles/blocks/_videopromo.scss]: what's going on here?

File: app/styles/blocks/_counters.scss (hunk 3 above)
Comment (vamidjo) [inline app/styles/blocks/_counters.scss]: does this go for `.dotcounter-item` as well?

Comment (xiwcx) [inline app/styles/blocks/_counters.scss]: goes for any and every class at the mortar level, but git blame says it was me who added all these!! my bad...

**Comment by xiwcx** [comment]
screen shot 2016-04-07 at 10 24 09 am

Looks like there are some color classes that need to get updated!

**Pull request merged**